unit-tests/func/playback/test-non-realtime.py

```diff
@@ -0,0 +1,31 @@
+# License: Apache 2.0. See LICENSE file in root directory.
```

Comment:
Matan put his record & playback test under func/rec-play/
Talk to him and synchronize please

Reply:
done